src/modules/platform/constants/NativeConstants.js

```diff
@@ -0,0 +1,15 @@
+// @flow
```

Comment:
If you want you can create a "native-constants" module in the `src/modules` directory as it is basically a standalone module.